cornice_swagger/swagger.py

```diff
@@ -276,14 +283,18 @@ class DefinitionHandler(object):
 
     json_pointer = '#/definitions/'
 
-    def __init__(self, ref=0):
+    def __init__(self, ref=0, typ_dispatcher=TypeConversionDispatcher()):
```

Comment:
The reason is that I needed to change the behaviour of the `TypeConversionDispatcher` and not only add some custom type to fit our use case. Since our custom types always inherits from colander.String, I've adopted a solution like this:

```python
class TypeConversionDispatcher(object):
    converters = defaultdict(lambda: StringTypeConverter)

    converters.update({
        colander.Boolean: BooleanTypeConverter,
        colander.Date: DateTypeConverter,
        colander.DateTime: DateTimeTypeConverter,
        colander.Float: NumberTypeConverter,
        colander.Integer: IntegerTypeConverter,
        colander.Mapping: ObjectTypeConverter,
        colander.Sequence: ArrayTypeConverter,
        colander.String: StringTypeConverter,
        colander.Time: TimeTypeConverter,
    })
...
```
I do realize that's not a common use case, so it's ok for me to go with a dict of type:converter